processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -480,42 +481,79 @@ public int compare(ByteBuffer lhsBuffer, ByteBuffer rhsBuffer, int lhsPosition,
       }
     }
 
-    @Override
-    public Comparator<Grouper.Entry<RowBasedKey>> entryComparator()
+    private Grouper.KeyComparator getBufferDimComparatorForNullFudgeTimestamp()
```

Comment:
Why not just a method for comparing ByteBuffers in `RowBasedGrouperHelper` and then calling this method from comparators, returned from `RowBasedKeySerde.comparator()`? Wrapping this functionality into another comparator object seems needless to me.